static/js/settings.js

```diff
@@ -821,6 +821,12 @@ exports.update_page = function () {
     i18n.ensure_i18n(_update_page);
 };
 
+exports.update_email = function () {
+    var message = i18n.t('We have sent your confirmation email.');
+    viewport.scrollTop(0);
```

Comment:
Also, this function being the thing triggered by `server_events.js` should not update the UI -- otherwise we'll update the UI on all open browsers, not just the one where the change was made...